wandb/sdk/artifacts/artifact.py

```diff
@@ -947,6 +947,7 @@ def _update(self) -> None:
                 ) {
                     artifact {
                         id
+                        ttlDurationSeconds
```

Comment:
Nit: let's be consistent in how we build the query.
```suggestion
                        _TTL_DURATION_SECONDS_FIELD_
```